Apollo/DeviceViewers/FadeViewer.xaml

```diff
@@ -1,24 +1,24 @@
-﻿<UserControl xmlns="https://github.com/avaloniaui"
-             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             x:Class="Apollo.DeviceViewers.FadeViewer"
-             xmlns:Components="clr-namespace:Apollo.Components"
-             Margin="10 5"
-             DetachedFromVisualTree="Unloaded">
+﻿<UserControl xmlns="https://github.com/avaloniaui" 
+  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" x:Class="Apollo.DeviceViewers.FadeViewer" 
+  xmlns:Components="clr-namespace:Apollo.Components" Margin="10 5" DetachedFromVisualTree="Unloaded">
+
+  <UserControl.Resources>
+    <SolidColorBrush x:Key="TitleBrush" />
+  </UserControl.Resources>
 
   <Grid ColumnDefinitions="Auto,Auto">
     <Grid Grid.Column="0" RowDefinitions="*,*">
       <Border Grid.Row="0" BorderBrush="{DynamicResource ThemeBorderHighBrush}" BorderThickness="0 0 0 1">
         <StackPanel Margin="0 7 0 0" VerticalAlignment="Center" Spacing="5">
           <Canvas Background="Transparent" Width="214" Height="26" x:Name="Canvas">
-            <Rectangle Canvas.Left="7" Canvas.Top="7" Width="200" Height="20" Stroke="{DynamicResource ThemeBorderHighBrush}" StrokeThickness="1" x:Name="Gradient"
-              PointerPressed="Canvas_MouseDown">
-              
+            <Rectangle Canvas.Left="7" Canvas.Top="7" Width="200" Height="20" Stroke="{DynamicResource ThemeBorderHighBrush}" StrokeThickness="1" x:Name="Gradient" PointerPressed="Canvas_MouseDown">
+
               <Rectangle.Fill>
                 <LinearGradientBrush StartPoint="0%,50%" EndPoint="100%,50%" />
```

Comment:
you could also update this gradient to reflect the fade type change

Reply:
It seems like the only way to do this would be to have more gradient stops to give the effect. Can you think of another way?

Reply:
I'm not sure, you should ask at https://gitter.im/AvaloniaUI/Avalonia one of the avalonia devs will probably point you in the right direction if there is a suitable way to modify the gradient. if there isn't, you can probably hack something "close enough" together with adding more points.

Reply:
I've done it, but I had to make some special adjustments for if the setting is 'Hold', might wanna check over it again to make sure it won't break. Was alright in my testing but who knows.